policies/policies.go

```diff
@@ -33,6 +33,34 @@ type Dataset struct {
 	Created      time.Time
 }
 
+type Page struct {
+	PageMetadata
+	Policies []Policy
+}
+
+type Service interface {
+	// CreatePolicy creates new agent Policy
+	AddPolicy(ctx context.Context, token string, p Policy, format string, policyData string) (Policy, error)
+
+	// RetrievePolicyByID retrieving policy by id with token
+	ViewPolicyByID(ctx context.Context, token string, policyID string) (Policy, error)
+
+	// RetrieveAll
+	ListPolicies(ctx context.Context, token string, pm PageMetadata) (Page, error)
+
+	// RetrievePolicyByIDInternal gRPC version of retrieving policy by id with no token
```

Comment:
change comments to match method names in this block

Reply:
Done